### PR TITLE
update geo-js to v0.28

### PR DESCRIPTION
**Pull request by oluademola**
# Pull Request

## What does this PR do?
**Update Geo-javascript Demo to v0.28**
- This update does not introduce any code-related breaking changes.
- Bumped instant-meilisearch to 0.8.0

## PR checklist
Please check if your PR fulfills the following requirements:
- [ ] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: src/geo-javascript/src/app.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ injectScript(
     const search = instantsearch({
       indexName: 'world_cities',
       searchClient: instantMeiliSearch(
-        'https://demos.meilisearch.com',
-        'L4h8HWpdac31f126b8eebbb23e2d10752e86d57d8a388e6910c2fedfb53542547dd4775c',
+        'https://ms-a79a247a550f-106.lon.meilisearch.io',
+        '87b5f3f9fd344ccad333adce78f21309fdecb4a72fa48608df1e9c6935d8c647',
         {
           limitPerRequest: 200,
```
Comment (bidoubiwa) [inline src/geo-javascript/src/app.js:15]: This parameter does not exists. It should be `paginationTotalHits: 200` and `finitePagination: true`

( I m noticing this error is also present in my playground haha)

Comment (oluademola) [reply src/geo-javascript/src/app.js]: Thanks 🙏

File: src/geo-javascript/src/app.js [hunk 2]
```diff
@@ -9,11 +9,8 @@ injectScript(
     const search = instantsearch({
       indexName: 'world_cities',
       searchClient: instantMeiliSearch(
-        'https://demos.meilisearch.com',
-        'L4h8HWpdac31f126b8eebbb23e2d10752e86d57d8a388e6910c2fedfb53542547dd4775c',
-        {
-          limitPerRequest: 200,
-        }
+        'https://ms-a79a247a550f-106.lon.meilisearch.io',
+        '87b5f3f9fd344ccad333adce78f21309fdecb4a72fa48608df1e9c6935d8c647'
```
Comment (bidoubiwa) [inline src/geo-javascript/src/app.js:13]:
```suggestion
        '87b5f3f9fd344ccad333adce78f21309fdecb4a72fa48608df1e9c6935d8c647',
```

we keep a trailing `,` for consistency

**Review by bidoubiwa — APPROVED** [review]
LGTM 🔥

**Review by CaroFG — APPROVED** [review]

**Pull request merged**